src/Library/demos/Box/main.css

```diff
@@ -0,0 +1,7 @@
+#card {
+  padding: 24px;
+}
+
+#border {
```

Comment:
This would make more sense as a CSS class, since you are adding and removing it programmatically. You can use `Gtk.Widget.add_css_class()` and `Gtk.Widget.remove_css_class()` for that.